client/landing/stepper/declarative-flow/internals/steps-repository/site-migration-credentials/hooks/use-credentials-form.ts

```diff
@@ -23,6 +26,18 @@ export const analyzeUrl = async ( domain: string ): Promise< UrlData | undefined
 	}
 };
 
+export const getHostingProvider = async ( domain: string ): Promise< string | undefined > => {
```

Comment:
We already have a [useHostingProviderQuery](https://github.com/Automattic/wp-calypso/blob/b2795a4b44e2f46b6c2ead43ac0d077f16191a30/client/data/site-profiler/use-hosting-provider-query.ts#L5) hook. Could we use that instead?